src/templates/react/Content/src/App.fs

```diff
@@ -66,11 +66,15 @@ let root model dispatch =
                     [ pageHtml model.currentPage ] ] ] ] ]
 
 open Elmish.React
+#if DEBUG
 open Elmish.Debug
```

Comment:
opening a module is not enough even in good-old F# on .NET: they act exactly as static class, i.e. static constructors and fields (and let bindings) are guaranteed to be initialized "at some point" before the *first use*... no use - no initialization!